File: src/main/java/de/hsa/bpmnandai/project/infrastructure/repository/BankLoanRequestRepository.java
package de.hsa.bpmnandai.project.infrastructure.repository;

import de.hsa.bpmnandai.project.infrastructure.BankLoanRequestEntity;
import org.springframework.data.jpa.repository.JpaRepository;

public interface BankLoanRequestRepository extends JpaRepository<BankLoanRequestEntity, String> {
}

File: src/main/kotlin/de/emaarco/bpmnandai/project/infrastructure/repository/BankLoanRequestRepository.kt
package de.emaarco.bpmnandai.project.infrastructure.repository

import org.springframework.data.jpa.repository.JpaRepository
import de.emaarco.bpmnandai.project.infrastructure.entity.BankLoanRequestEntity

interface BankLoanRequestRepository : JpaRepository<BankLoanRequestEntity, String> {

    fun findByRequestId(id: String): BankLoanRequestEntity?

}